ajax/widget/js/widget.js

```diff
@@ -0,0 +1,41 @@
+let employeeList = new XMLHttpRequest();
+employeeList.onreadystatechange = function () {
+    if (employeeList.readyState === 4) {
+        let employees = JSON.parse(employeeList.responseText);
+        let employeeStatus = '<ul class="bulleted">';
+        for (let i = 0; i < employees.length; i++) {
+            if (employees[i].inoffice === true) {
+                employeeStatus += '<li class="in">';
+            } else {
+                employeeStatus += '<li class="out">';
+            }
+            employeeStatus += employees[i].name;
+            employeeStatus += '</li>';
```

Comment:
What about setting employeeStatus to `in` or `out` and then building employeeStatus entirely below the conditional?

```
           let employeeStatusClass = '';
           if (employees[i].inoffice === true) {
                employeeStatusClass = 'in';
            } else {
                employeeStatusClass = 'out';
            }
            employeeStatus += `<li class="${employeeStatusClass}">`
            employeeStatus += employees[i].name;
            employeeStatus += '</li>';
```

You could also do this where you set a default value up front and only change it if the conditional is met.
```
           let employeeStatusClass = 'out'
           if (employees[i].inoffice === true) {
                employeeStatusClass = 'in';
            }
            employeeStatus += `<li class="${employeeStatusClass}">`
            employeeStatus += employees[i].name;
            employeeStatus += '</li>';
```

Reply:
Done. I liked the default status idea most. Also inlined all 3 `employeeStatus` assignments into one template literal.